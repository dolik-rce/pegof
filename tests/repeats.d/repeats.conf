input repeats.d/repeats.peg
optimize
