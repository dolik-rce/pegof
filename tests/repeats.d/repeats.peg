A1 <- X X

B1 <- X+ X

C1 <- X* X

A2 <- X X+

B2 <- X+ X+

C2 <- X* X+

A3 <- X X*

B3 <- X+ X*

C3 <- X* X*
