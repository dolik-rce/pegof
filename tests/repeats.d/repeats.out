A1 <- X X

B1 <- X+ X

C1 <- X+

A2 <- X X+

B2 <- X+ X+

C2 <- X+

A3 <- X+

B3 <- X+

C3 <- X*
